source/slang/slang-emit-c-like.cpp

```diff
@@ -214,6 +220,8 @@ void CLikeSourceEmitter::emitSimpleType(IRType* type)
 
         case kIROp_FloatType:   return UnownedStringSlice("float");    
         case kIROp_DoubleType:  return UnownedStringSlice("double");
+
+        case kIROp_CharType:    return UnownedStringSlice("uint8_t");
```

Comment:
We probably also need to have a design discussion and some written rules about what we want for the whole menagerie of 8-bit integer types. This is especially true in a world where interop with C/C++ probably requires us to be able to interact with functions that use any of `char`, `unsigned char`, and `signed char`, knowing that there are no explicit guarantees about the signed-ness of `char`. Of course that leaves aside the way that most of the time `char` is being used to represent a *byte* rather than a character, and `uint8_t`/`int8_t` don't do that significantly better.

We need to pick some terminology here and then zealously enforce it throughout the compiler stack or we are going to end up with a mess of well-intentioned but broken types.